js/migration/components/intro.js

```diff
@@ -16,22 +17,24 @@ import { __ } from '@wordpress/i18n';
  * @return {React.ReactElement} The introduction to the migration.
  */
 const Intro = () => {
+	// @ts-ignore
+	const editorGifUrl = blockLabMigration.editorGifUrl;
 	const developerNoticeUrl = 'https://getblocklab.com/migrating-to-genesis-custom-blocks/';
-	const announcementUrl = 'https://getblocklab.com/the-block-lab-team-are-joining-wp-engine/';
 
 	return (
 		<>
 			<div>
 				<h1>{ __( 'Migrating to Genesis Custom Blocks', 'block-lab' ) }</h1>
-				<p>{ __( 'In April, the Block Lab team joined the Genesis team at WP Engine. With our full-time focus, we’re very excited about the future of custom block tooling in WordPress. You can read more about that moment in this', 'block-lab' ) } <a target="_blank" rel="noopener noreferrer" className="text-purple-600 underline hover:text-purple-700" href={ announcementUrl }>{ __( 'announcement post', 'block-lab' ) }.</a></p>
-				<p>
-					{ __( 'As part of this move, we have been working on a new plugin that is based on what we developed at Block Lab.', 'block-lab' ) }
-					&nbsp;
-					{ __( 'Genesis Custom Blocks is now the home of all our custom block efforts and what we have planned is very very cool!', 'block-lab' ) }
-					&nbsp;
-					{ __( 'Version 1.0 of this plugin is now released and has full feature parity with Block Lab.', 'block-lab' ) }
-				</p>
-				<p>{ __( 'To continue receiving the best of what our team is building, we encourage you to migrate over. Our migration tool makes this nice and easy, and for the majority of use cases, completely automated.', 'block-lab' ) }</p>
+				<p>{ __( 'We’ve been busy! Since the Block Lab team joined WP Engine in 2020, the new plugin (Genesis Custom Blocks) has got a whole lot better. Plenty of things under the hood, but the biggest change is the new Block Builder interface.', 'block-lab' ) }</p>
+				<p>{ __( 'Check it out below!', 'block-lab' ) }</p>
+				<img
+					width={ 800 }
+					height={ 534 }
+					src={ editorGifUrl }
+					alt={ __( 'New editor screencast', 'block-lab' ) }
+				/>
+				<p>{ __( 'Migrating from Block Lab to Genesis Custom Blocks is super easy, seamless, and (for 95% of users) completely automated.', 'block-lab' ) }</p>
```

Comment:
Just curious about this 95% number. Is that accurate?

Reply:
Good question. The 5% would probably be either people that have used PHP filters or where there is a bug. The PHP filters aren't that common.

Reply:
Do you think 'the vast majority' would convey this better?